scalafmt-core/js/src/main/scala/org/scalafmt/internal/package.scala

```diff
@@ -0,0 +1,5 @@
+package org.scalafmt
+
+package object internal {
+  type PriorityQueue[T] = scala.collection.mutable.PriorityQueue[T]
```

Comment:
What does this get used for?  isn't the `new PriorityQueue[State]()` trying to make a new version of the new class?

Reply:
This is used so that the type `org.scalafmt.internal.PriorityQueue[T]` is an alias for `scala.collection.mutable.PriorityQueue[T]`. That is, I wanted to make a type synonym, so that the shared code does not break when searching for an `internal.PriorityQueue`, but in Java the PriorityQueue is reimplemented to use the `java.util` one.